docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3.7'
+services:
+  
+  cdp4_db:
+    image: rheagroup/cdp4-database-community-edition:latest
+    hostname: cdp4-postgresql
+    command: postgres -c max_locks_per_transaction=1024
+    environment:
+      - POSTGRES_PASSWORD=${DB_POSTGRESPASSWORD}
+      - POSTGRES_USER=postgres
+    networks:
+      cdp4:
+        aliases:
+          - cdp4-postgresql
+    container_name: cdp4-database-community-edition
+    restart: always
+    ports:
+      - '${DB_HOSTPORT}:5432'
+    expose:
+      - '5432'
+    volumes:
+      - cdpdbdata:/var/lib/postgresql/data
+      - cdpdblogs:/logs
+
+  cdp4_webservices:
+    hostname: cdp4-webservices
+    restart: always
+    build: .
+    networks:
+      cdp4:
+        aliases:
+          - cdp4-webservices
+    depends_on:
+      - cdp4_db
+    ports:
+      - "${WEBSERVICES_HOSTPORT}:5000"
+    expose:
+      - '5000'
```

Comment:
what happens when we host multiple cdp4 instances on the same host machine?

Reply:
This scenario still needs to be explored. In principle it is ok, but the correct DB port still needs to be injected somehow in the correct service. But we need to make a task for this soon,

Reply:
hm.. ok... please add a task for that now so we do not forget. This is rather important to be able to do, otherwise we will have to stick with bare-metal installs ourselves

Reply:
https://github.com/RHEAGROUP/CDP4-Docker-Compose/issues/2